takwimu/takwimu_ui/src/components/Hero.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+import { Button, Link, Grid, Typography, withStyles } from '@material-ui/core';
+
+import Section from './Section';
+
+import africanParliament from '../assets/images/africanparliament.jpg';
+import triangle from '../assets/images/triangle.svg';
+
+const styles = theme => ({
+  section: {
+    marginTop: '2.25rem',
+
+    // On mobile, ensure we don't have unnecessary horizontal scroll
+    overflow: 'hidden'
```

Comment:
Why do w have overflow if everything fits?

Reply:
Nice catch, that was before I enabled `word-break` on title @karimkawambwa. Removing it.